pkg/services/accesscontrol/models.go

```diff
@@ -319,6 +319,9 @@ const (
 	ActionTeamsWrite            = "teams:write"
 	ActionTeamsPermissionsRead  = "teams.permissions:read"
 	ActionTeamsPermissionsWrite = "teams.permissions:write"
+
+	// Team related scopes
+	ScopeTeamsAll = "teams:*"
```

Comment:
What @gamab said - I think we should try to use the `Scope` function for defining scopes.